addon/setup-links.js

```diff
@@ -27,7 +27,18 @@ function getFieldDefinitionSetupLinks(sfHost, fieldName, fieldDefinition, isCust
 }
 
 export async function getFieldSetupLinks(sfHost, sobjectName, fieldName, isCustomSetting) {
-  let {records: fieldDefinitions} = await sfConn.rest(`/services/data/v${apiVersion}/tooling/query/?q=${encodeURIComponent(`select DurableId from FieldDefinition where EntityDefinition.QualifiedApiName = '${sobjectName}' and QualifiedApiName = '${fieldName}'`)}`);
+  let qualifiedApiName = fieldName;
+  let suffixOfAddressFields = ["City", "CountryCode", "GeocodeAccuracy", "Latitude", "Longitude", "PostalCode", "StateCode", "Street"];
+  let replaceQualifiedName = isCustomSetting ? "__c" : "Address";

```

Comment:
isCustomSetting is different from custom field or custom object. To detect if field must be translate check if 
["City", "CountryCode", "GeocodeAccuracy", "Latitude", "Longitude", "PostalCode", "StateCode", "Street", "__City__s","__CountryCode__s",...]
for (let suffixOfAddressField of suffixOfAddressFields) {
    if (fieldName.endsWith(suffixOfAddressField)) {// maybe do tolowercase to be sure
      let replaceQualifiedName = fieldName.endsWith("__s") ? "__c" : "Address";
      qualifiedApiName = qualifiedApiName.replace(suffixOfAddressField, replaceQualifiedName);
    }
  }